<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Concesionarios mi pana</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(229, 229, 229);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="crearButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CREAR VEHICULO</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QCheckBox" name="motorCheckBox">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Motor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSlider" name="motorSlider">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>80</number>
       </property>
       <property name="maximum">
        <number>450</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="motorLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Potencia del motor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QSpinBox" name="motorSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>80</number>
       </property>
       <property name="maximum">
        <number>450</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>711</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,0">
     <item row="0" column="1">
      <widget class="QRadioButton" name="ruedaDeRepuestoButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rueda de repuesto</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">grupoComplemento</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="kitDePinchazosButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Reparacion de pinchazos</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">grupoComplemento</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="complementoLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Complemento (Elegir una)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="1">
      <widget class="QLabel" name="matriculaLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Matricula</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="generarMatriculaButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Generar placa</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="matriculaTextViewer">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>711</width>
      <height>156</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <widget class="QComboBox" name="combustibleComboBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Sin combustible</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eléctrico</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diesel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gasolina</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Híbrido</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Queroseno</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="ruedasComboBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="colorComboBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentText">
        <string>Blanco</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Blanco</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Negro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rojo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Azul</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Amarillo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Verde</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="colorLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nombreVehiculoLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nombreLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nombre del vehículo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ruedasLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Número de ruedas</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="gasolinaLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Combustible</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="vehiculosCreadosLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Vehiculos creados</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vehiculosCreadosTextViewer">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="tituloLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>CREA TU PROPIO VEHICULO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="1">
      <widget class="QCheckBox" name="reactoresCheckBox">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Reactores</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="alasCheckBox">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Alas</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="trenDeAterrizajeCheckBox">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tren de aterrizaje</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="locomotoraCheckBox">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Locomotora</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="vagonesLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Vagones</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="vagonesSpinBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QListWidget" name="vehiculosCreadosListWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="grupoComplemento"/>
 </buttongroups>
</ui>
